context/design-tokens/build-component.js

```diff
@@ -18,40 +18,61 @@ function getStyleDictionaryConfig(brand, components) {
 
 	return {
 		include: [
-			`${__dirname}/literal/**/*.json`,
-			`${__dirname}/alias/**/*.json`
+			`${__dirname}/literal/default/**/*.json`,
+			`${__dirname}/literal/${brand}/**/*.json`,
+			`${__dirname}/alias/default/**/*.json`,
+			`${__dirname}/alias/${brand}/**/*.json`
 		],
 		source: [
-			`${__dirname}/component/**/*.json`
+			`${__dirname}/components/${brand}/**/*.json`
 		],
 		platforms: {
 			scssVariables: {
 				transformGroup: 'web',
 				buildPath: `${dest}/`,
 				files: components.map(component => {
 					return {
-						destination: `${component}/scss/10-settings/_${component}.variables.scss`,
+						destination: `${component}/scss/00-tokens/_${component}.tokens.scss`,
 						format: 'scss/variables',
 						filter: tokenFilter(brand, component),
 						"options": {
-							"outputReferences": true
+							"outputReferences": true,
+							"showFileHeader": false
 						}
 					}
 				})
 			}
 		}
 	}
 }
-// TODO: see if I can combine this with an existing Sass file
 
 console.log('Build started...');
 
-['global'].map(function (brand) {
-	let dir = `${__dirname}/component/${brand}`
+// the following array needs to be manually updated when more brand speicifc components are added to brands not on this list. Otherwise this will not build. This is why it is currently not part of
+['springer'].map(function (brand) {
```

Comment:
Does this means that so far `global-forms`, if not all design-tokens aware components, only supports the springer brand?

Sorry if this question sounds weird, I've no experience so far with design-token and have too less of an overview of this big rehaul work

Reply:
this might be my lack of JS skills -- but if I include all brands but a `.json` file for that that brand doesn't exist - the process ends. 

So, I'm aware of this, and in lieu of better JavaScript skills I'll add brands as and when brand specific component tokens are available :)